.chloggen/replace-create-rbac-permissions-by-checking-the-sa-permissions.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Automatically enable RBAC creation if operator SA can create clusterroles and bindings. We should also add link to the manifest that defines the RBAC. --create-rbac-permissions flag is noop and deprecated now.
```

Comment:
`We should also add link to the manifest that defines the RBAC` should not be there. I was asking to add a link to the RBAC users can create for the operator SA

Reply:
Sorry.I don't know how I commited this.

Reply:
The problem with the link is that we will duplicate the information because we are applying it to `kustomize` too. And we need to apply the patches as I did in this PR because otherwise the PolicyRules list will be overridden.